File: samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.IsSeparator/CS/isseparator.cs
// <snippet10>
using System;

public class IsSeparatorSample {
	public static void Main() {
		string str = "twain1 twain2";

		Console.WriteLine(char.IsSeparator('a'));		// Output: "False"
		Console.WriteLine(char.IsSeparator(str, 6));	// Output: "True"
	}
}
// </snippet10>

File: snippets/fsharp/VS_Snippets_CLR_System/system.Char.IsSeparator/FS/isseparator.fs
module isseparator

// <snippet10>
open System

let str = "twain1 twain2"

printfn $"{Char.IsSeparator 'a'}"       // Output: "False"
printfn $"{Char.IsSeparator(str, 6)}"   // Output: "True"

// </snippet10>
